domain/modeldefaults/service/providers.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/juju/errors"
+	"github.com/juju/utils/v3/ssh"
+
+	"github.com/juju/juju/environs/config"
+)
+
+// ControllerModelConfigService represents a slimmed down model config service
+// for providing access to the controller models configuration.
```

Comment:
apostrophe

Reply:
This file should never have made it in. Done & Done